packages/core/src/utils.ts

```diff
@@ -27,3 +27,31 @@ export const rewriteRedirectTo = (
           : options?.redirectTo
       }
     : options
+
+export function getParameterByName(name: string, url?: string) {
+  if (!url) url = window?.location?.href || ''
+  // eslint-disable-next-line no-useless-escape
+  name = name.replace(/[\[\]]/g, '\\$&')
+  const regex = new RegExp('[?&#]' + name + '(=([^&#]*)|&|#|$)'),
+    results = regex.exec(url)
+  if (!results) return null
+  if (!results[2]) return ''
+  return decodeURIComponent(results[2].replace(/\+/g, ' '))
+}
+
+export function removeParameterFromWindow(name: string) {
+  const location = window?.location
```

Comment:
This will break in a Node environment. We should return if `window` is of the type `undefined`.
```ts
if (typeof window === 'undefined') {
  return
}

// ...
const { location } = window
```